src/App.js

```diff
@@ -1,29 +1,6 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
+import './components/Calculator.css';
 
```

Comment:
- [ ] I cannot load the CSS file as the file name is totally different it is `calculator.css` not `Calculator.css`. Please fix this issue as well.